recipes/zserio/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake
+
+class ZserioTestPackageConanFile(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
```

Comment:
```suggestion
    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv", "VirtualBuildEnv"
```

You exported `ZSERIO_JAR_FILE` to build_env, but did not say that you need it here. It should fix your error with Macos.

Reply:
trying now